src/DataService/DataObjectFactory.php

```diff
@@ -2,6 +2,7 @@
 
 namespace Lullabot\Mpx\DataService;
 
+use GuzzleHttp\Promise\PromiseInterface;
```

Comment:
You can ignore the changes in this file as I am already improving them in a later PR.